tests/run_all_tests.py

```diff
@@ -0,0 +1,7 @@
+import os 
+
+print('=========================== running unit tests ========================')
+os.system('testflo unit_tests -v')
```

Comment:
I think the default flags we want to use is probably something like `-v -n 4`, since otherwise testflo treats logical cores as physical cores. This would be consistent with the previous `mpirun -np 4` command in `run_reg_tests.py`.